core/src/main/java/org/apache/iceberg/SnapshotSummary.java

```diff
@@ -354,4 +354,8 @@ private static void setIf(boolean expression, ImmutableMap.Builder<String, Strin
       builder.put(property, String.valueOf(value));
     }
   }
+
+  private static void set(ImmutableMap.Builder<String, String> builder, String property, Object value) {
```

Comment:
What's the value of this method instead of `builder.set(property, value)`?

Reply:
That's a good point. Thanks